pubsub/iam.js

```diff
@@ -170,60 +204,80 @@ function testSubscriptionPermissionsExample (subscriptionName, callback) {
     'pubsub.subscriptions.update'
   ];
 
-  // Retrieve the IAM policy for the provided subscription
-  subscription.iam.testPermissions(
-    tests,
-    function (err, permissions, apiResponse) {
-      if (err) {
-        return callback(err);
-      }
-
-      console.log('Got permissions for ' + subscriptionName);
-      callback(null, permissions, apiResponse);
+  // Test the IAM policy for the specified subscription
+  subscription.iam.testPermissions(tests, function (err, permissions) {
```

Comment:
`permissions`, but I will rename to `permissionsToTest` so as not to shadow the `permissions` variable that appears later